### PR TITLE
Igvf-261-add-button (renamed branch)

### PR DESCRIPTION
**Pull request by dougli1sqrd**
I had to create a new branch name in order to satisfy the demo gods (underscores not allowed in branch names). Since PRs don't know how to track changed branch names, here's a new PR with the same code. Here is the old PR: https://github.com/IGVF-DACC/igvf-ui/pull/82

### CODE REVIEW

File: components/add.js [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    );
+  }
+};
+
+AddLink.propTypes = {
+  collection: PropTypes.object.isRequired,
+  label: PropTypes.string,
+  type: PropTypes.string,
+  size: PropTypes.string,
+};
```
Comment (forresttanaka) [inline components/add.js:107]: It’s not required, but you can catch more errors is you select the allowed values with [PropTypes.oneOf](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes)

**Review by forresttanaka — DISMISSED** [review]
Odd you have to “run prettier.” It should run itself every time you save your file.

**Review by forresttanaka — DISMISSED** [review]

**Review by forresttanaka — CHANGES_REQUESTED** [review]
OK, now _I’m_ going to complain about a lack of gaps. There’s no gap between the Add button and the panel on individual schema pages. If you sign in and look at the Pages page, you can see an Add Page button with an `mb-1` gap between it and the panel. Could you add that gap to the schema add button as well? It’s also a different color, but I don’t know if I should make the “Add Page” button primary-outline or if you should make the Add button success-outline. We haven’t decided on button colors and styles overall yet.

**Comment by dougli1sqrd** [comment]
I had to fiddle a little, but I got it

**Review by forresttanaka — APPROVED** [review]
Thanks. We need to get back to work on the design document so that similar things on different pages appear the same.

**Pull request merged**